integration-test/src/test/java/org/jboss/pnc/integration_new/endpoint/BuildTest.java

```diff
@@ -309,10 +309,10 @@ private Boolean groupBuildToFinish(String groupBuildId, EnumSet<BuildStatus> isI
             logger.debug("Gotten build with status: {}", build.getStatus());
             if (!build.getStatus().isFinal()) return false;
         } catch (RemoteResourceNotFoundException e) {
-            //expected
+            logger.error("Group Build with id:{} not present", groupBuildId);
```

Comment:
The test should FAIL immediately in this case.

Reply:
sure